IRP/src/CommonModules/Localization/Module.bsl

```diff
@@ -1185,6 +1196,8 @@ Function Strings(Lang) Export
 	
 	Strings.Insert("InfoMessage_034", NStr("en = 'Time zone not changed'", Lang));
 	Strings.Insert("InfoMessage_035", NStr("en = 'Time zone changed to %1'", Lang));
+	Strings.Insert("InfoMessage_036", NStr("en = 'New document movements are no different from manual corrections. The ""Manual movements edit"" checkbox can be removed'", Lang));
```

Comment:
"New document movements are identical to the manual corrections. The "Manual Movements Edit" checkbox is now unnecessary and can be removed."